docs/contracts/function-modifiers.rst

```diff
@@ -21,7 +21,7 @@ if they are marked ``virtual``. For details, please see
     pragma solidity >0.7.0 <0.9.0;
 
     contract owned {
-        constructor() { owner = msg.sender; }
+        constructor() { owner = payable(msg.sender); }
```

Comment:
Not needed.

Reply:
Actually it is needed in this case, but I'm wondering if owner needs to be payable.